release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -293,6 +293,68 @@ This dashboard provides API server metrics, such as:
 
 // TODO: Link to troubleshooting using this dashboard section once it is available
 
+[id="ocp-4-7-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.7 includes the following alerting rule changes:
```

Comment:
4.7 also added
```
AlertmanagerClusterCrashlooping
AlertmanagerClusterDown
AlertmanagerClusterFailedToSendAlerts
AlertmanagerFailedToSendAlerts

KubeletClientCertificateRenewalErrors
KubeletServerCertificateRenewalErrors

NTODegraded
NTOPodsNotReady

PrometheusOperatorNotReady
PrometheusOperatorRejectedResources
PrometheusOperatorSyncFailed
PrometheusTargetLimitHit
```
removed since 4.7
```etcdHighNumberOfFailedHTTPRequests```


47 rules see from
https://drive.google.com/file/d/1gjXXMnEsEKP2bCD1X3DZ_-W-K9f25Ci-/view?usp=sharing

46 rules see from
https://drive.google.com/file/d/1TXYR7qn10nh84gM-A866R_JdOoDNgD6d/view?usp=sharing

Reply:
@juzhao @simonpasquier PTAL at the additional new alerts added (see juzhao's previous comment). I also included the removed `etcdHighNumberOfFailedHTTPRequests` alerts (warning and critical).